readthedocs/proxito/urls.py

```diff
@@ -44,8 +45,26 @@
 from readthedocs.proxito.views.redirects import redirect_page_with_filename
 from readthedocs.proxito.views.utils import fast_404
 
+DOC_PATH_PREFIX = getattr(settings, 'DOC_PATH_PREFIX', '')
 
 urlpatterns = [
+    # Serve downloads
+    url(
+        (
+            # /_/downloads/<lang>/<ver>/<type>/
+            # /_/downloads/<alias>/<lang>/<ver>/<type>/
+            r'^{DOC_PATH_PREFIX}downloads/'
+            r'(?:(?P<subproject_slug>{project_slug})/)?'
```

Comment:
Should we just have 2 different URLConfs for this, instead of an optional regex? Feels cleaner to me.